darts/models/forecasting/block_rnn_model.py

```diff
@@ -183,6 +185,61 @@ def forward(self, x_in: Tuple):
         # predictions is of size (batch_size, output_chunk_length, 1)
         return predictions
 
+    def _rnn_sequence(
+        self,
+        name: str,
+        input_size: int,
+        hidden_dim: int,
+        num_layers: int,
+        dropout: float = 0.0,
+        normalization: str = None,
+    ):
+
+        modules = []
+        is_lstm = self.name == "LSTM"
+        for i in range(num_layers):
+            input = input_size if (i == 0) else hidden_dim
+            is_last = i == num_layers - 1
+            rnn = getattr(nn, name)(input, hidden_dim, 1, batch_first=True)
+
+            modules.append(rnn)
+            modules.append(ExtractRnnOutput(not is_last, is_lstm))
+            modules.append(nn.Dropout(dropout))
+            if normalization:
+                modules.append(self._normalization_layer(normalization, hidden_dim))
+            if not is_last:  # pytorch RNNs don't have dropout applied on the last layer
+                modules.append(nn.Dropout(dropout))
+
+        return nn.Sequential(*modules)
+
+    def _fc_layer(
+        self,
+        input_size: int,
+        num_layers_out_fc: list[int],
+        target_size: int,
+        normalization: str = None,
+    ):
+        if not num_layers_out_fc:
+            num_layers_out_fc = []
+
+        last = input_size
+        feats = []
+        for feature in num_layers_out_fc + [
+            self.output_chunk_length * target_size * self.nr_params
+        ]:
+            if normalization:
+                feats.append(self._normalization_layer(normalization, last))
+            feats.append(nn.Linear(last, feature))
+            last = feature
+        return nn.Sequential(*feats)
+
+    def _normalization_layer(self, normalization: str, hidden_size: int):
```

Comment:
if `normalization` is different from `batch` and `layer` the method return `None`. is this intended?